recipes/openmathreasoning/prompts/genselect.yaml

```diff
@@ -0,0 +1,49 @@
+
```

Comment:
what's the difference between this and what's in main prompts folder?

Reply:
Oops forgot to remove it. Had kept it in the `recipes` folder, but then decided to move it to `nemo_skills/prompt`. Removing it now